app/package.json

```diff
@@ -18,6 +18,6 @@
   },
   "homepage": "https://zmk.dev/",
   "devDependencies": {
-    "prettier": "^2.4.0"
+    "prettier": "^2.7.1"
```

Comment:
Please revert these unrelated changes.

Reply:
strange... my local version says only 2.7.1

Reply:
:shrug: Should be able to use `git log --stat` to find which commit changed it, and fix it up w/ an interactive rebase.